.Rbuildignore

```diff
@@ -17,3 +17,5 @@
 ^README\.md\.in$
 ^pkgapi_.*\.tar\.gz$
 ^\.travis\.yml$
+^codecov\.yml$
+^\.covrignore$
```

Comment:
Useful to add `*.Rproj`, or is there a good reason for not doing that? I have `sircovid.Rproj` which I'd prefer to be ignored

Reply:
that should be added to the Rbuildignore, yeah.  Elsewhere I have

```
^.*\.Rproj$
^\.Rproj\.user$
```

for Rstudio files.  I'll add them now, but we should commit the project file too - could you push it to this branch please?

Reply:
Will do, then I will merge this